### PR TITLE
Added CHANGELOG-1.2.md for the v1.2.0 release

### PR DESCRIPTION
**Pull request by lintongj**
**What this PR does / why we need it**: Added CHANGELOG-1.2.md for the v1.2.0 release

**Which issue(s) this PR fixes**: 

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

**Review by xinyanw409 — APPROVED** [review]
LGTM.

File: changelogs/CHANGELOG-1.2.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Changelog since v1.1.0
+
+## v1.2.0
+
+Date: TBD
+
+### Changes
+
+- Added the check of restic annotation in vsphere plugin to skip pvc snapshot for restic annotated volumes ([#349](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/349), [@lintongj](https://github.com/lintongj))
+- Fix to determine cluster type from vsphere-csi-controller deployment
```
Comment (xing-yang) [inline changelogs/CHANGELOG-1.2.md:10]: Need a period at the end.

File: changelogs/CHANGELOG-1.2.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  Fix is to watch for vc config secret in vmware-system-csi namespace for vanilla setups.
+  The secret location is the same for wcp setups, no change is necessary.
+  A restart of backup-driver and data-manager is necessary when csi driver is upgraded to 2.3. ([#348](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/348), [@deepakkinni](https://github.com/deepakkinni))
```
Comment (xing-yang) [inline changelogs/CHANGELOG-1.2.md:13]: Restart is only needed for vanilla?

Comment (lintongj) [reply changelogs/CHANGELOG-1.2.md:13]: @deepakkinni Would you please propose a change log entry for your PR #348? Looks like it is kind of confusing.

File: changelogs/CHANGELOG-1.2.md (hunk 1 above)
Comment (xing-yang) [inline changelogs/CHANGELOG-1.2.md:10]: This line can be removed.

File: changelogs/CHANGELOG-1.2.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Changelog since v1.1.0
+
+## v1.2.0
+
+Date: TBD
+
+### Changes
+
+- Added the check of restic annotation in vsphere plugin to skip pvc snapshot for restic annotated volumes. ([#349](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/349), [@lintongj](https://github.com/lintongj))
+- Added CSI 2.3 support in plugin by updating the way to determine cluster flavor and determine the secret to watch. ([#348](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/348), [@deepakkinni](https://github.com/deepakkinni))
```
Comment (xing-yang) [inline changelogs/CHANGELOG-1.2.md:10]: CSI -> vSphere CSI Driver

**Review by xing-yang — APPROVED** [review]
LGTM

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/562/

**Pull request merged**